pyglet/window/__init__.py

```diff
@@ -1133,7 +1133,7 @@ def maximize(self):
         """
         raise NotImplementedError('abstract')
 
-    def set_vsync(self, vsync):
+    def set_vsync(self, vsync: bool) -> None:
```

Comment:
Not sure if this should default to True maybe?

Reply:
I'd probably leave it as-is, just to force users to be explicit. 

However, it _might_ make sense to just turn some of these things into `@property`s instead of using the `set_xxxx` methods. (Since it's for v2.0 it's a chance to modernize a bit).  What do you think?

Reply:
I've thought about it and I'm thinking that I would like to clean the code up first, then look at making many of these into properties

Reply:
I'm thinking that things like this and mouse_exclusive/keyboard_exclusive would be prime candidates for properties

Reply:
That sounds reasonable.